src/bika/ui/jbot/senaite.core.browser.viewlets.templates.sections.pt

```diff
@@ -0,0 +1,38 @@
+<tal:sections
+  xmlns:tal="http://xml.zope.org/namespaces/tal"
+  xmlns:metal="http://xml.zope.org/namespaces/metal"
+  xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+  tal:define="portal_tabs view/portal_tabs;
+            portal_state context/@@plone_portal_state;
+            portal_url portal_state/portal_url;"
+  tal:condition="portal_tabs"
+  i18n:domain="plone">
+
+  <nav class="dropdown">
+    <a href="#"
+       class="nav-link dropdown-toggle"
+       data-toggle="dropdown"
+       aria-haspopup="true"
+       aria-expanded="false">
+       <img width="24px" src="" alt="icon" tal:attributes="src string:${portal_url}/++plone++bika.ui.static/assets/icons/outcome.png"/>
```

Comment:
Please remove width="24px", rather use an icon that's closest to the required size

Reply:
Using a lower quality image will lead to the icon being washed out after a bit of zooming in. Compare the Blank icon (smaller image)  vs the Control icon (bigger image). Is there a reason why there shouldn't be a width?
![image](https://user-images.githubusercontent.com/104898641/190455398-6352759e-5049-4847-870a-ff77e767a8ef.png)

Reply:
It is just good practice to avoid inline styling and to keep things the same as done on other places on the code. I think maybe you can resize the image or ask Lemoene to give you the image that has been resized to the size that you want.